src/style.css

```
@@ -1,3 +1,67 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color-accent: #00ACDC;
+}
+
+html {
+  color: #000;
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  color: #000;
+  text-transform: uppercase;
+  text-decoration: none;
+  position: relative;
+}
+
+.logo__img {
+  margin: 10px 0;
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link.is-active,
```

Don't increase specificity
```suggestion
.is-active,
```